### ISSUE TITLE
Image as background

### ISSUE THREAD

**Issue by dennissergeev** [pyvista/pyvista-support#91]
## Description

I'm probably being too lazy, but I haven't found any example of setting a background of `Plotter` to an image. Is it possible at all? `Plotter.set_background()` seems to work only with colours.

Thanks.

**Comment by banesullivan** [comment]
Nope, you’re not being lazy! we don’t have this implemented at the moment. 

Intermixing the news code in this example could do it http://kwwidgets.org/Wiki/VTK/Examples/Python/BackgroundImage

Also note this requires layered rendering which is a very stale PR: https://github.com/pyvista/pyvista/pull/367

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by dennissergeev** [comment]
Thank you for the quick response!
The VTK example seems a bit intimidating, but I'll give it a go. It would be great to have it implemented in pyvista of course...
